src/components/FileInput.js

```diff
@@ -20,21 +22,25 @@ export default class FileInput extends Component {
     this.fileInputId = `file-input-${fileCounter++}`;
   }
 
-  onFileChange(event) {
-    const fileList = Array.from(event.target.files);
+  onFileSelection() {
+    let options = {properties: ['openfile']}
```

Comment:
```suggestion
    let options = {properties: ['openFile']}
```
This should be `openFile`, capital F, or else no files will be selectable (on osx at least).